implementations/index.md

```diff
@@ -57,6 +57,7 @@ isomorphic ActiveRecord clone that issues JSON:API requests instead of SQL and i
 
 ### <a href="#client-libraries-typescript" id="client-libraries-typescript" class="headerlink"></a> Typescript
 * [ts-angular-jsonapi](https://github.com/reyesoft/ts-angular-jsonapi) A JSON:API library developed for AngularJS in Typescript
+* [nestjs-json-api](https://github.com/ringcentral/nestjs-json-api) Tools to implement JSON API, such as, end point, query params, body params, validation and transformation response.
```

Comment:
I think it would be helpful if the short description mentions the stack for which it implements the JSON:API specification. If I got it right, the library is for Nest.js, isn't it?

Reply:
yes. I have added a need stack for using it.